data/transactions/logic/README.md

```diff
@@ -479,60 +505,60 @@ Some of these have immediate data in the byte or bytes after the opcode.
 ##### Scalar Fields
 | Index | Name | Type | In | Notes |
 | - | ------ | -- | - | --------- |
-| 0 | Sender | []byte |      | 32 byte address |
+| 0 | Sender | addr |      | 32 byte address |
```

Comment:
I've held back on this, as it seems nitpicking, but want your thoughts: why `addr` instead of `address`? If this is about documentation, why not go all the way?

Reply:
fair enough, would prevent it from being somehow mistaken for the pseudo-op

Reply:
I think addr has become mixed-use due to AuthAddr